src/wp-content/themes/twentytwenty/functions.php

```diff
@@ -304,7 +304,7 @@ function twentytwenty_menus() {
  * @since Twenty Twenty 1.0
  *
  * @param string $html The HTML output from get_custom_logo() (core function).
- * @return string
+ * @return string|false Custom logo HTML or false if not available.
```

Comment:
Sorry, my suggestion before was incorrect. I was looking at `get_theme_mod()` which returns `false` when the theme mod is missing. But here, when the theme mod is missing, it just passes the `$html` string straight through.

```suggestion
 * @return string Filtered custom logo HTML.
```

Reply:
Thanks @westonruter, I’ve updated it with your corrected suggestion.